packages/office-ui-fabric-react/src/components/Dropdown/Dropdown.tsx

```diff
@@ -91,7 +91,7 @@ export class Dropdown extends BaseComponent<IDropdownProps, IDropdownState> {
           aria-expanded={ isOpen ? 'true' : 'false' }
           role='combobox'
           aria-live={ disabled || isOpen ? 'off' : 'assertive' }
-          aria-label={ label }
+          aria-label={ ariaLabel }
```

Comment:
Yeah, I did think of it while making the change, but label and ariaLabel were split not just to decouple them, but also to be consistent with other components. It would sense to fall back to label, so that we wont be broken, _but then we wouldn't we consistent with other components_. So I didnt do it.